### PR TITLE
Update openCV and Protobuf

### PR DESCRIPTION
**Pull request by klecki**
Update dependencies to latest versions.

Source snapshot for FFmpeg was adjusted.

Signed-off-by: Krzysztof Lecki <klecki@nvidia.com>

### CODE REVIEW

**Comment by klecki** [comment]
Builds:
2282811, 2282810

**Comment by klecki** [comment]
Builds:
2283144, 2283145

File: build_scripts/build_ffmpeg.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ if [ ${WITH_FFMPEG} -gt 0 ]; then
         --disable-os2threads \
         --disable-dct \
         --disable-dwt \
-        --disable-error-resilience \
+        --enable-error-resilience \
```
Comment (jantonguirao) [inline build_scripts/build_ffmpeg.sh:33]: wondering, why that change?

**Review by jantonguirao — APPROVED** [review]

**Review by JanuszL — APPROVED** [review]

**Comment by klecki** [comment]
Builds: 2286805, 2286806

**Pull request merged**